src/data/Pager/Pager.tsx

```diff
@@ -123,7 +133,7 @@ export const Pager: FC<PagerProps> = ({
     }
   }, [canNext, page, onPageChange, pageCount]);
 
-  if (pageCount === 1) {
+  if (hideOnSinglePage && pageCount === 1) {
```

Comment:
I think it would be better to just remove this logic and the user can deteremine if they want to do this themselves. If you just want to remove this check, we can push a release for ya.

Reply:
>I think it would be better to just remove this logic and the user can deteremine if they want to do this themselves. 
This makes sense.
>If you just want to remove this check, we can push a release for ya.
There is an ask to show the pagination at all times, so that would be great 👍

Reply:
Let me know what you want me to do here.

Reply:
Ya, just delete that if statement/additions and lgtm.